content/articles/adding-permission-to-flutter-apps/index.md

```diff
@@ -0,0 +1,130 @@
+
+### Adding permission to flutter applications
+### Introduction
+The first important thing to know is that the android system uses permissions. You can for example for your app to be able to use the internet or access the camera.
+So you may not want to let your user's friends download the app on their device if it is not necessary.
+The same, you may not want your user's apps to use your camera if they are not necessary.
+This article is about how to add permission when you create an app with flutter (if you want). It's simple but there are some things to know to avoid.  Your camera usage without permission, for example.
+
+I will write my application so that the user has to accept permissions when he wants them. Users have an option to see all installed apps and select which one can see your camera.
+(To know if the user accepted the permissions I am going to use a new flutter thing called StreamBuilder. If you do not know what it is, never mind for now).
+What's important here is that when I get this app installed on my device. I will see a screen with two buttons, one to 'accept permissions' and the other to 'dismiss'. This is not what you will do in your app but it's to show how StreamBuilder works.
+### Prerequisite 
```

Comment:
These don't sound like prerequisites.
Please follow this doc [here](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md) for guidance.